### PR TITLE
Multiple Deletion of Resources

### PR DESCRIPTION
**Pull request by thayashi**
Support Multiple Deletion, inspired by @nati's PR.

### CODE REVIEW

**Comment by thayashi** [comment]
It looks like:
screen shot 2016-11-17 at 1 09 17 pm

**Comment by nati** [comment]
LGTM

File: app/templates/table.html [hunk 1]
```diff
@@ -22,6 +36,13 @@
 %>
           </select>
           <input type="text" class="form-control"  placeholder="Search" value="<%= searchQuery.value %>">
+<%
+          if (selectedResource > 0) {
+%>
+          <a class="btn btn-danger" data-gohan="delete-all">Delete All</a>
```
Comment (stepek) [inline app/templates/table.html]: It shouldn't be a "Delete Selected" ?

File: app/js/views/tableView.js [hunk 2]
```diff
@@ -367,6 +430,12 @@ export default class TableView extends View {
       }
       return result;
     });
+    let uiState = this.collection.map(model => {
```
Comment (stepek) [inline app/js/views/tableView.js]: it should be const

File: app/templates/table.html (hunk 1 above)
Comment (thayashi) [inline app/templates/table.html]: Fixed

File: app/js/views/tableView.js (hunk 2 above)
Comment (thayashi) [inline app/js/views/tableView.js]: Fixed

**Pull request merged**